Resources/Private/Partials/Backend/SharedApplicationActions/Show.html

```diff
@@ -3,8 +3,12 @@
 
 <div class="row">
 	<div class="col-md-6 summary-block table-container">
-		<h3><f:translate id="be.label.ApplicationSingle.Overview.PersonalData"></f:translate></h3>
-
+		<h3>
+			<f:translate id="be.label.ApplicationSingle.Overview.PersonalData"></f:translate>
+			<f:link.action action="show" target="_blank" arguments="{application: application, print: 1}" class="print">
+				<f:image src="EXT:ats/Resources/Public/Icons/btn_print.png" alt="print" />
```

Comment:
Please use `<img src="{f:uri.resource()}">` instead, otherwise you'll get additional `sys_file` entries which won't be updated if you change e.g. image dimensions.